draft-beck-tls-trust-anchor-ids.md

```diff
@@ -470,39 +470,43 @@ The above does not apply if the authenticating party is a client. This protocol
 
 # Security Considerations
 
-## Relying Party Policies
+## Incorrect Selection Metadata
+
+If the authenticating party has incorrect information about trust anchor identifiers, it may send an untrusted certification path. This will not result in that path being trusted, but the connection will likely fail.
```

Comment:
```suggestion
If the authenticating party has provisioned certification paths with incorrect trust anchor identifier information, it may inadvertently send untrusted certification paths when matching against this identifier and additionally may overlook this certification path when attempting to match against the correct identifier. In either case, this will not result in that path becoming trusted, but the connection will fail when this leads to the selection of an untrusted certification path.
```